Document/0x06b-Basic-Security-Testing.md

```diff
@@ -660,12 +660,17 @@ Note: when you use Clutch on iOS 12, please check [Clutch Github issue 228](http
 
 ###### Using Frida-ios-dump
 
-[Frida-ios-dump](https://github.com/AloneMonkey/frida-ios-dump "Frida-ios-dump") requires Frida server running on your jailbroken device. It is basically using Frida scripts to dump the decrypted binary from memory onto a file.
-Note: before we get started, please note that the Frida-ios-dump tool is not always compatible with the latest version of Frida. Therefore: it might well be that you have to install an older version of Frida on your jailbroken device.
-First, make sure that the configuration in `dump.py` is set to either localhost with port 2222 when using iProxy, or to the actual IP address and port of the device from which you want to dump the binary. Next, change the username and password to the ones you use. Now you can safely use the tool to enumerate the apps installed:
+[Frida-ios-dump](https://github.com/AloneMonkey/frida-ios-dump "Frida-ios-dump") requires Frida server running on your jailbroken device. It is basically using Frida script (`dump.js`) to dump the decrypted binary from memory onto a file.
+
+> Frida-ios-dump might not be compatible with the latest version of Frida. If that's the case, you might have to install an older version of Frida server on your device.
```

Comment:
Same not about tools being potentially broken. I would say, unless there is very specific <version or  >version info, we shouldn't mention it.

Reply:
Agree. We shouldn't maintain this kind of things. It should be of common sense to take a look in the repo for (known) issues and for creating them if something fails.